packages/smooth_app/lib/l10n/app_ar.arb

```diff
@@ -722,7 +723,7 @@
             "count": {}
         }
     },
-    "plural_compare_x_products": "{count,plural, zero {يقارن {count} منتجات} two {يقارن {count} منتجات} few {يقارن {count} منتجات} many {يقارن {count} منتجات}=1{قارن منتج واحد} other{يقارن {count} منتجات}}",
+    "plural_compare_x_products": "{count,plural, zero {} one {يقارن {count} منتجات} two {يقارن {count} منتجات} few {يقارن {count} منتجات} many {يقارن {count} منتجات}=1{قارن منتج واحد} other{يقارن {count} منتجات}}",
```

Comment:
@monsieurtanuki I guess that's not the desired translation, right ? If so, I'm going to delete them all in Crowdin.

Reply:
@teolemon I know there's been some glitches in the translations starting about a month ago (flutter 3.7?), like additional tags (`one` when we already had `1=`).
I tried to solve some cases recently in #3841 in some `app.arb` files, but we should probably do something upstream too (crowdin) and I may have been clumsy in RTL languages (`ar`, `he`).